File: HW6_OOP/src/model/Note.java
package model;

public class Note {
    private String id;
    private String title;
    private String content;
    private String date;

    public Note(String id, String title, String content, String date) {
        this.id = id;
        this.title = title;
        this.content = content;
        this.date = date;
    }

    public String getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getContent() {
        return content;
    }

    public String getDate() {
        return date;
    }


    public void setTitle(String title) {
        this.title = title;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public void setDate(String date) {
        this.date = date;
    }

    @Override
    public String toString() {
        return String.format("%s / %s /  %s / %s", this.getId(), this.getTitle(), this.getContent(), this.getDate());
    }
}

File: NotesPythonAttestat/model/note.py
import datetime


class Note:
    def __init__(self, id: int, title: str, content: str, date: str):
        self.id = id
        self.title = title
        self.content = content
        self.date = date

    def get_id(self):
        return self.id

    def get_title(self):
        return self.title

    def get_content(self):
        return self.content

    def get_date(self):
        return self.date

    def set_new_date(self):
        self.date = datetime.datetime.now()

    def set_new_title(self, new_title: str):
        self.title = new_title

    def set_new_content(self, new_content: str):
        self.content = new_content
